eo.json

```diff
@@ -1284,35 +1284,35 @@
         "Tutorial.Grab.EquipForFullUse": "Ekipu uzi ĉiujn funkciojn",
 
         "Tutorial.WorldOrb.Click": "Alklaki Mondan Globon",
-        "Tutorial.WorldOrb.Info": "<b>Monda Globo</b> \n<size=80%> Reprezentas mondon aŭ sesion</size>\nDuobla Alklako - Eniri \nSola Klako - Kunteksta Menuo",
+        "Tutorial.WorldOrb.Info": "<b>Monda Globo</b> \n<size=80%> Reprezentas mondon aŭ seancon</size>\nDuobla Alklako - Eniri \nSola Klako - Kunteksta Menuo",
 
         "Tutorial.Tooltip.GeneralPrimary": "Uzi",
         "Tutorial.Tooltip.GeneralSecondary": "Malĉefa Ago",
         "Tutorial.Tooltip.ContextMenu" : "Aliri ilmenuo kaj malekipa opcion",
         "Tutorial.Tooltip.DevPrimary": "Interagi kun aparatoj (teni)",
-        "Tutorial.Tooltip.DevSecondary": "Elektu (montras aparatojn)",
+        "Tutorial.Tooltip.DevSecondary": "Elekti (montras aparatojn)",
         "Tutorial.Tooltip.Duplicator": "Multobligo (dum tenado de ero)",
         "Tutorial.Tooltip.GluePrimary": "Apliki Gluon (teni)",
         "Tutorial.Tooltip.GlueSecondary": "Devigi gluon sekiĝi",
         "Tutorial.Tooltip.ColorPrimary": "Apliki koloron",
-        "Tutorial.Tooltip.ColorSecondary": "Elektu Koloron",
+        "Tutorial.Tooltip.ColorSecondary": "Elekti Koloron",
         "Tutorial.Tooltip.Brush": "Desegni",
         "Tutorial.Tooltip.MaterialPrimary": "Apliki Materialon \n<size=75%> (Metu materialan okulkavon en la pafilon) </size>",
-        "Tutorial.Tooltip.MaterialSecondary": "Ĉerpu Materialon \n<size=75%> (Certigu, ke unue ne ekzistas materiala okulkavo) </size>",
+        "Tutorial.Tooltip.MaterialSecondary": "Ĉerpi Materialon \n<size=75%> (Certigu, ke unue ne ekzistas materiala okulkavo) </size>",
         "Tutorial.Tooltip.MeshPrimary": "Anstataŭigi Maŝon",
         "Tutorial.Tooltip.MeshSecondary": "Ĉerpi Maŝon",
         "Tutorial.Tooltip.CharacterColliderPrimary": "Aldoni Karakteran Kolizion",
         "Tutorial.Tooltip.CharacterColliderSecondary": "Forigi Karakteran Kolizion",
-        "Tutorial.Tooltip.GrabbableSetterPrimary": "Faru Grabbable",
-        "Tutorial.Tooltip.GrabbableSetterSecondary": "Faru Non-Grabbable",
+        "Tutorial.Tooltip.GrabbableSetterPrimary": "Farigi Tenieble",
+        "Tutorial.Tooltip.GrabbableSetterSecondary": "Farigi Maltenieble",
         "Tutorial.Tooltip.LightPrimary": "Krei Lumon (tenu)",
         "Tutorial.Tooltip.LightSecondary": "Ŝanĝi Koloron",
         "Tutorial.Tooltip.MeshVisibilityPrimary": "Vidigi",
         "Tutorial.Tooltip.MeshVisibilitySecondary": "Fari Nevideblan",
         "Tutorial.Tooltip.ShapePrimary": "Krei Formon",
         "Tutorial.Tooltip.ShapeSecondary": "Ŝanĝi Formon",
-        "Tutorial.Tooltip.WigglerPrimary": "Ellasu Nesprimitajn Malbonojn Sur La Mondon",
-        "Tutorial.Tooltip.WigglerSecondary": "Elaĉetu vian animon iomete",
+        "Tutorial.Tooltip.WigglerPrimary": "Ellasi Nesprimitajn Malbonojn Sur La Mondon",
```

Comment:
Here, and in the next line, I think the implicit mode was originally imperative, so it should be -u. In other locales it is also translated as imperative.
And the second word should be "Neesprimitajn", with double e